apps/strapi-cms/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Creating multi-stage build for production
+FROM node:18-alpine as build
```

Comment:
you should get a version using a SHA.
Moreover, I think you should set the version to a variable and use the very same version (avoiding duplication) also in the second layer of this multi-stage build.

I would use the version defined in the [`.node-version` file](https://github.com/pagopa/b2b-portals/blob/main/.node-version)

Reply:
ok, with last commit i have insert var and SHA by new row:  "ARG NODE_IMAGE=node:18-alpine@sha256:d51f2f5ce2dc7dfcc27fc2aa27a6edc66f6b89825ed4c7249ed0a7298c20a45a"
The node version defined in the [.node-version file](https://github.com/pagopa/b2b-portals/blob/main/.node-version) is not presente in DockerHub image: https://hub.docker.com/_/node/tags?page=1&name=18.7.1
The image "node:18.7-alpine" has multiple vulnerabilities, strapi recommends using the "node:18-alpine" image.
Furthermore, inserting the SHA we cannot bind the docker file to the node version defined in the [.node-version file](https://github.com/pagopa/b2b-portals/blob/main/.node-version)